### DISCUSSION TITLE
Calendar javascript configuration problem

### DISCUSSION THREAD

**Discussion by marcmaalouly** [Laravel-Backpack/community-forum#962] [Q&A (Help)] [answered]
Hello,

Firstly i want to start by thanking @pxpm for this amazing package.

I need some help with the package related with deployment.
I did some changes in the javascript configuration using the following code in the setupCalendarOperation function

`CRUD::setOperationSetting('javascript-configuration', [
            'headerToolbar' => [
                'start' => 'title',
                'center' => '',
                'end' => 'dayGridMonth timeGridWeek timeGridDay',
            ],
            'timeZone' => 'UTC'
        ]);`

this is working normally on my local machine. But once i deploy its like its not reading those configuration. The header tool bar stays as default and the timezone keeps changing to the user locale.
I tried clearing all caches but still nothing.

Any help?

**Comment by marcmaalouly** [comment]
<img width="1139" alt="Screenshot 2024-04-24 at 5 21 51 PM" src="https://github.com/Laravel-Backpack/community-forum/assets/65512028/7210c617-9642-43d6-b9c8-faa7e61a3d38">
this is locally and the correct configuration.


And below is how its showing on the live server
<img width="1139" alt="Screenshot 2024-04-24 at 5 21 55 PM" src="https://github.com/Laravel-Backpack/community-forum/assets/65512028/2b2f2d72-18a2-4998-a661-470e7375aa18">

**Comment by jcastroa87** [comment, marked as answer]
Hello @marcmaalouly 

Did you try a

```
php artisan optimize:clear
php artisan basset:clear
```

Let me know if that works for you.

Cheers.

**Reply by marcmaalouly** [reply]
Thank you @jcastroa87 . php artisan basset:clear fixed the problem

**Comment by jcastroa87** [comment]
Im glad that works for you.

Cheers.
